### PR TITLE
Posts for 2023 Git courses

### PR DESCRIPTION
**Pull request by ns-rse**
Two events pages and a blog post on the Git courses scheduled for next year.

### CODE REVIEW

File: 2022-12-20-upcoming-git-zero-to-hero.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (davidwyld) [inline 2022-12-20-upcoming-git-zero-to-hero.md]: This file should be in `_posts/`.

File: _events/workshop-2023-01-26-git-zero-hero.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: "git & GitHub through GitKraken - from Zero to Hero!"
+category: gitzerohero
+tags: collaboration, git, github, gitkraken
+permalink: /training/workshop/2022-11-28-git-zero-hero
```
Comment (davidwyld) [inline _events/workshop-2023-01-26-git-zero-hero.md]:
```suggestion
permalink: /training/workshop/2023-01-26-git-zero-hero
```

File: _events/workshop-2023-03-06-git-zero-hero.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: "git & GitHub through GitKraken - from Zero to Hero!"
+category: gitzerohero
+tags: collaboration, git, github, gitkraken
+permalink: /training/workshop/2022-11-28-git-zero-hero
```
Comment (davidwyld) [inline _events/workshop-2023-03-06-git-zero-hero.md]:
```suggestion
permalink: /training/workshop/2023-03-06-git-zero-hero
```

**Review by davidwyld — CHANGES_REQUESTED** [review]
:+1: Nice work getting all this together!
Couple of minor corrections.

Also, would it be a good idea to add to the training posts something like:
> Please note this workshop will be held **in person** - if you would like to attend an online course instead, please see...

and *vice versa*?

File: 2022-12-20-upcoming-git-zero-to-hero.md (hunk 1 above)
Comment (ns-rse) [reply 2022-12-20-upcoming-git-zero-to-hero.md]: That is what I get for working before my brain is awake, sorry and thanks.

**Review by davidwyld — APPROVED** [review]
I hope you don't mind me taking the liberty of moving the blog post to the `_posts/` folder (for blog md files), sorry if this is incorrect.

File: 2022-12-20-upcoming-git-zero-to-hero.md (hunk 1 above)
Comment (davidwyld) [reply 2022-12-20-upcoming-git-zero-to-hero.md]: Don't apologise please, that's why we have reviews!

**Comment by ns-rse** [comment]
Not at all, I thought I did that...

```
❱ history | grep "git mv" | grep 2022-12-20
15619  2022-12-20 12:42  git mv 2022-12-20-upcoming-git-zero-to-hero.md _events/.
```

Not sure why it didn't come through when I staged and pushed. :shrug:

**Comment by davidwyld** [comment]
> Not at all, I thought I did that...
> 
> ```
> ❱ history | grep "git mv" | grep 2022-12-20
> 15619  2022-12-20 12:42  git mv 2022-12-20-upcoming-git-zero-to-hero.md _events/.
> ```
> 
> Not sure why it didn't come through when I staged and pushed. shrug

You moved it to `_events/` rather than `_posts/` assuming that this is a blog post and not an event.

**Comment by ns-rse** [comment]
Gah, I'm an idiot! :facepalm:

**Pull request merged**